docs/release-notes/release-notes-0.15.1.md

```diff
@@ -20,24 +20,29 @@
 
 * [Add previous_outpoints to listchaintxns](https://github.com/lightningnetwork/lnd/pull/6321)
 
+* [Add wallet reserve rpc & field in wallet balance](https://github.com/lightningnetwork/lnd/pull/6592)
```

Comment:
This cleanup is already being done in https://github.com/lightningnetwork/lnd/pull/6685, which we'll merge soon.

Reply:
Gotcha! Reverted.